apps/aechannel/src/aesc_state_cache.erl

```diff
@@ -78,8 +78,10 @@
              min_depth = ?MIN_DEPTH}).
 -type ch_cache_id() :: { aeser_id:val()
                        , aeser_id:val() | atom()}.
+-type opts() :: map().
 -record(ch_cache, { cache_id    :: ch_cache_id() | undefined
                   , state       :: aesc_offchain_state:state() | atom() | undefined
+                  , opts = #{}  :: map() | '_'
```

Comment:
Why the catch all?

Reply:
It's used in a select pattern, and Dialyzer gets cranky otherwise. :)